systest.py

```diff
@@ -295,6 +296,31 @@ def assert_not_equal(self, first, second):
                                                   repr(first),
                                                   repr(second)))
 
+    def assert_multiline_equal(self, first, second):
```

Comment:
I'm not sure, but I think I prefer `assert_text_equal()` instead of `assert_multiline_equal()`. What do you think?

Reply:
I wasn't quite ready to pull the trigger on this merge request but since you've taken the time to have a look...

I'm not wedded to "multiline". It describes in very technical terms what the assertion is for but "text" is perhaps more obvious. I am happy to change it.

Reply:
Ok, let's rename it to *text* then.

In general the PR looks good. Other users will certainly find the new assertion useful. There is some code styling issues, but I can fix those after merging.